lms/djangoapps/instructor_task/tasks.py

```diff
@@ -4,7 +4,7 @@
 
 At present, these tasks all operate on StudentModule objects in one way or another,
 so they share a visitor architecture.  Each task defines an "update function" that
-takes a module_descriptor, a particular StudentModule object, and xblock_instance_args.
+takes a module_block, a particular StudentModule object, and xblock_instance_args.
```

Comment:
Why do we want to use `module_block` instead of just `block` here and in `lms/djangoapps/instructor_task/tasks_helper/module_state.py`?

Reply:
I initially thought that might be needed for retaining context. I've updated it to just `block` now.